modules/core/include/opencv2/core/matx.hpp

```diff
@@ -391,6 +391,8 @@ template<typename _Tp, int cn> class Vec : public Matx<_Tp, cn, 1>
     const _Tp& operator ()(int i) const;
     _Tp& operator ()(int i);
 
+    constexpr Vec<_Tp, cn>& operator=(const Vec<_Tp, cn>& rhs) = default;
```

Comment:
1. `#ifdef CV_CXX11` (due `= default`)
2. Do we really need `constexpr` here? if so, please use `CV_CONSTEXPR` instead. IMHO, it is better to remove `contexpr` here completely (see Windows builder warnings)
3. What is about move assignment operator (`Vec<_Tp, cn>&& rhs`)?

Reply:
According to:
http://eel.is/c++draft/dcl.fct.def.default#3
> If a function is explicitly defaulted on its first declaration, it is implicitly considered to be constexpr if the implicit declaration would be.

As it *is* implicit constexpr, the "constexpr" can be removed without loss for the relevant cases (you can of course have a Vec<Foo, 1>, where Foo would lead to non-constexpr).

move assignments and constructors are implicitly "deleted" already, as there is a user-declared copy constructor. Both would be pointless for an array on the stack anyway (you can not efficiently move/swap these, not anymore than a memcpy would do).